p5/maze-generation/sketch.js

```diff
@@ -44,7 +44,9 @@ function draw() {
             const x = col * scl;
             const y = row * scl;
             noStroke();
-            if (cell.visited) {
+            if (current[0] == col && current[1] == row) {
```

Comment:
could you make this small change to fix the error when coming back to the first cell, else it breaks when ending the maze

```suggestion
            if (current && current[0] == col && current[1] == row) {
```